### DISCUSSION TITLE
Mask Of The Vestals

### DISCUSSION THREAD

**Discussion by TomekPL-1** [secretcowlevel/DoomtrooperPublic#18] [Card Balance]
Mask Of The Vestals (BROTHERHOOD) [UR]
EQUIPMENT, UNIQUE, FACTION
BROTHERHOOD recipient only. During your turn, if you Annihilate an ART card from your hand, the next two cards that the Warrior's controller plays from his hand he may draw a card to replace it. Discarded cards are not replaced in this manner. If you use Mask of the Vestals until beginning of  your next turn your hand cannot be affected by other cards owned by you.

**Comment by TomekPL-1** [comment]
Or 21-ric suggestion little bit updated
Mask Of The Vestals (BROTHERHOOD) [UR]
EQUIPMENT, UNIQUE, FACTION
BROTHERHOOD recipient only. During your turn (ecxept during combat), if you cast an ART card draw three cards, then annihilate two cards from your hand. If this is second ART played this turn you may annihilate only one card.

**Comment by Kite80** [comment]
I'd prefer to stick more with the original, making it less OP of course. I like the idea of cards annihilated to pay the cost, so I came up with a mechanism like MTG's brainstorm:

Mask Of The Vestals (BROTHERHOOD) [UR]
EQUIPMENT, UNIQUE, FACTION
BROTHERHOOD recipient only. 1 charge counter (max 1 counters). During your turn, if you play a card from your hand you can choose to spend 1 charge counter and draw 3 cards from your deck, then choose one of them and annihilate the rest. You can spend 1 standard action to recharge the Mask Of The Vestals.
